web-admin/src/routes/[organization]/[project]/+layout.svelte

```diff
@@ -21,6 +24,14 @@
   }
 
   $: onProjectPage = isProjectPage($page);
+
+  $: if ($page.params.project && $user.data?.user?.id) {
+    metricsService.loadCloudFields({
+      isDev: window.location.host.startsWith("localhost"),
+      projectId: $page.params.project,
+      userId: $user.data?.user?.id,
+    });
+  }
```

Comment:
1. What about tracking the `organizationId` too?
2. Can we load the `userId` from the `AvatarButton.svelte` component? Because the userID will be present if the user is logged in on the Home and Org pages.